index.js

```diff
@@ -555,7 +555,12 @@ MediaSession.prototype = extend(MediaSession.prototype, {
             this._log('info', 'sending source remove', filteredDesc);
             this._log('info', 'sending source add', filteredNewDesc);
```

Comment:
should this log statement be moved down inside the setTimeout block?

Reply:
ya